src/middleware.ts

```diff
@@ -61,6 +61,10 @@ type Redirect = {
 };
 
 const REDIRECTS: Redirect[] = [
+  {
+    from: '/platforms/javascript/guides/react/configuration/integrations/trycatch/',
```

Comment:
Is it only react? Or should we make this dynamic for all guides?

Reply:
I think it makes sense to make it dynamic it applied to other guides before